README.md

```diff
@@ -56,3 +56,11 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing
+A number of unit tests have been added in /tests folder. They have been written with pytest in mind.
+Before unit testing make sure to have a task recently completed to be able to trigger all the tests.
```

Comment:
Tests should not depend on the state of the real board. Only selenium tests should be making requests to Trello and even then it should have its own board rather than depending on "real" data.

See my other comment on changing the test_app.py tests into unit tests.